docs/self-hosting/docker.md

```diff
@@ -17,96 +17,107 @@ hide_title: false
+   $ git clone --single-branch --branch main https://github.com/standardnotes/standalone.git
+   $ cd standalone
    ```
 
-1. Install `git` and `libmysqlclient-dev`:
+1. Initialize default configuration files by typing:
 
    ```bash
-   $ sudo apt-get install -y git libmysqlclient-dev
+   $ ./server.sh setup
    ```
 
-1. Make sure you are in your home directory and clone the Standard Notes Server [syncing-server](https://github.com/standardnotes/syncing-server) project:
+1. Customize your configuration
+
+   In the `.env` file you will find 3 variables that need to be filled in with generated secret keys. The variables are `JWT_SECRET`, `LEGACY_JWT_SECRET`, `AUTH_JWT_SECRET`. You can generate their values using:
 
    ```bash
-   $ cd ~
-   $ git clone --single-branch --branch master https://github.com/standardnotes/syncing-server.git
-   $ cd syncing-server
+   $ openssl rand -hex 32
    ```
 
-> **Note:** The `master` branch has the latest, stable code. Use this branch in production environments.
+   Use the same method to change the value of `ENCRYPTION_SERVER_KEY` in `docker/auth.env` file
+
+  > **Note** Environment variables cannot be changed with effect to take place while the docker containers are running. To change them, server needs to be restarted.
 
-1. Create `.env` file in the project's directory:
+1. (Optional) Customize the port
+
+  By default the syncing server will run on port 3000. If it is the case that you have a different service running on that port you can customize the port on which you want to run the infrastructure on. To do that, edit the `EXPOSED_PORT` variable in the `.env` file.
+
+1. Simply run:
 
    ```bash
-   $ cp .env.sample .env
+   $ ./server.sh start
    ```
 
-   Generate the secret key base and paste the output in the `.env` file's `SECRET_KEY_BASE` and `PSEUDO_KEY_PARAMS_KEY` key:
+   This should load all the microservices that the infrastructure consists of.
+
+  > **Note** First run might take a while since there are Docker images to be pulled and built and migrations initializing the database to be run.
+
+1. Wait for the infrastructure to bootstrap
+
+   It takes a moment for the infrastructure to bootstrap and all the microservices to start. You can observe the process by typing:
 
    ```bash
-   $ openssl rand -hex 64
+   $ ./server.sh logs
    ```
 
-   Also make sure to check the following variables:
+  > **Note** You can safely escape from logs with CTRL+C
 
-   - `DB_PASSWORD` create a password for your database and set it
-   - `RAILS_ENV` change this to "production" for production use, otherwise the access token time is very short and forces re-login
+  > **Note** Microservices depend on each other and because of that they are starting sequentially in our setup. In the logs you will most probably observe that one service is waiting for another to start with lines like this: "XYZ is unavailable yet - waiting for it to start" where XYZ is the dependant service name. That is expected.
```

Comment:
```suggestion
  > **Note** Microservices depend on each other and because of that they are starting sequentially in our setup. In the logs you will most probably observe that one service is waiting for another to start with lines like: "XYZ is unavailable yet - waiting for it to start" where XYZ is the dependent service name. This is expected.
```